Mathlib/Algebra/Category/Ring/Adjunctions.lean

```diff
@@ -91,6 +91,9 @@ instance : HasForget₂ CommRingCat CommMonCat where
   forget₂ := { obj M := .of M, map f := CommMonCat.ofHom f.hom }
   forget_comp := rfl
 
+#adaptation_note
+/-- `simp` is taking longer after nightly-2025-03-25. -/
```

Comment:
```suggestion
#adaptation_note /-- nightly-2025-03-25
`simp` is taking longer after nightly-2025-03-25. -/
```
This way it will be somewhat easier to track/classify adaptation notes using `grep`.